src/conftest.py

```diff
@@ -0,0 +1 @@
+# Keep empty, this tells pytest to test here
```

Comment:
Please could you show why this file is needed? What error do you get without it? I'm able to test without it.

Reply:
Oh, I added this so that pytest knew where to test. I must have had a weird version, or something. Let me see if it's necessary,

Reply:
Removed